x-pack/platform/plugins/shared/cases/server/client/cases/get.ts

```diff
@@ -281,6 +286,7 @@ export const resolve = async (
         savedObject: resolvedSavedObject,
         comments: theComments.saved_objects,
         totalComment: theComments.total,
+        totalEvents: countEventsForID({ comments: theComments }),
         totalAlerts: countAlertsForID({ comments: theComments, id: resolvedSavedObject.id }),
```

Comment:
I wonder if we it makes sense to introduce aggregations for this instead of counting them on the backend. Or do the alert and comment counts in a single run instead of two that are executed in sequence. Wdyt?

Reply:
> in sequence as in how its done right now?

Yes, right now it's in sequence but it could be done in a single `reduce`

Reply:
makes sense but I dont want to make this change  here - how about I create a separate issue for this and assign it to myself?